File: python_ruby/Logical/1.rb
puts("아이디를 입력하세요.")
input_id = gets.chomp()

puts("비밀번호를 입력하세요.")
input_pw = gets.chomp()

real_id = "egoing"
real_pw = "1111"

if real_id == input_id
  if real_pw == input_pw
    puts("hello, egoing")
  elsif
    puts("wrong password")
  end
else
  puts("wrong ID")
end

File: python_ruby/Logical/1.py
str_id = input("아이디를 입력하세요.")
str_pw = input("패드를 입력하세요.")

real_id = "egoing"
real_pw = "1111"

if str_id == real_id:
    if str_pw == real_pw:
        print("hello, egoing!")
    else:
        print("wrong password")
else:
    print("wrong id")
